src/ui/core/zcl_abapgit_gui_css_processor.clas.testclasses.abap

```diff
@@ -101,6 +70,7 @@ CLASS ltcl_single_file IMPLEMENTATION.
   ENDMETHOD.
 
   METHOD test_empty_file.
+    " TODO REMOVE ?? why test empty files ?
```

Comment:
Why not? Test all the things.

Reply:
Well, yes, point is that real assetman actually **fails** when data is empty (opposite to the double).
So my statement is inaccurate. Probably the test should not be removed but changed to fail if no exception happened ... I'll fix

Reply:
Ah okay. Then it makes sense to align these two.
I just didn't expect the asset manager to fail if an asset is empty.

Reply:
I've adjusted in the last commit

Reply:
> I just didn't expect the asset manager to fail if an asset is empty.

Yeap, I actually have a doubt if this is correct at all :( On the other hand what is the point of empty assets ? Anyway let's reconsider it when there a real issue